engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/IOContextTest.scala

```diff
@@ -0,0 +1,86 @@
+package org.enso.interpreter.test.semantic
+
+import org.enso.interpreter.test.{InterpreterContext, InterpreterTest}
+import org.graalvm.polyglot.Context
+
+class DevelopmentContextTest extends InterpreterTest {
+
+  override def contextModifiers: Option[Context#Builder => Context#Builder] =
+    Some(_.option("enso.IOEnvironment", "development"))
```

Comment:
Is this the reason why these tests need to be in Scala and cannot be just pure Enso tests? Can we get a comment explaining that?